apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -15,11 +15,17 @@ const noAssignment = assignmentId(null, null);
 const decideLater = '__decideLater__';
 const isValidAssignment = id => id !== noAssignment && id !== decideLater;
 
+const hasAssignmentFamily = (assignmentFamilies, assignment)  => (
+  assignment && !!assignmentFamilies.find(assignmentFamily => (
```

Comment:
Nit: [`array.some(predicate)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) would be more idiomatic here than `!!array.find(predicate)`.